seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/java/org/apache/seatunnel/e2e/connector/file/local/LocalFileIT.java

```diff
@@ -224,6 +248,14 @@ public class LocalFileIT extends TestSuiteBase {
                         "/seatunnel/read/zip/excel/multifile/multiZip.zip",
                         container);
 
+                Path xlsxGz =
+                        convertToGzFile(
+                                Lists.newArrayList(
+                                        ContainerUtil.getResourcesFile("/excel/e2e.xlsx")),
+                                "e2e-xlsx-gz");
```

Comment:
After testing, it seems that it is a compression problem. ？xlsx file cannot be converted after compression？

Reply:
@zhdech remember to adjust doc if you remove .xlsx support

Reply:
We CAN NOT remove .xlsx support.

Reply:
Even after removing Excel, it still cannot pass.
I suspect there may be some issues with the original case